<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="INBook_X1" failures="1" tests="1" name="edgetest" time="40.346" errors="0" timestamp="2024-03-16T09:39:16 IST">
  <testcase classname="testCases.TC_02_LoginTest" name="test1" time="25.056">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_02_LoginTest.test1(TC_02_LoginTest.java:66)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
</testsuite> <!-- edgetest -->
